frame/system/src/lib.rs

```diff
@@ -606,12 +609,14 @@ pub mod pallet {
 	/// Events have a large in-memory size. Box the events to not go out-of-memory
 	/// just in case someone still reads them from within the runtime.
 	#[pallet::storage]
+	#[benchmarking(cached)]
 	#[pallet::unbounded]
 	pub(super) type Events<T: Config> =
 		StorageValue<_, Vec<Box<EventRecord<T::Event, T::Hash>>>, ValueQuery>;
 
 	/// The number of events in the `Events<T>` list.
 	#[pallet::storage]
+	#[benchmarking(cached)]
```

Comment:
I'm a bit ambivalent about this syntax, `benchmarking(cached)` seems to indicate that I can put keywords other than `cached` within the parenthesis. Is that true?

Reply:
i also do not have a strong opinion about syntax, this syntax was suggested here: https://github.com/paritytech/substrate/issues/10251